WordPress/Classes/ViewRelated/Me/App Settings/Boolean User Defaults/BooleanUserDefaultsDebugViewModel.swift

```diff
@@ -16,40 +16,51 @@ final class BooleanUserDefaultsDebugViewModel: ObservableObject {
         }
     }
 
-    @Published var userDefaultsSections: Sections = []
+    @MainActor @Published private(set) var userDefaultsSections: Sections = []
```

Comment:
Decorating this property with `@MainActor` means that it must be mutated on the main thread.